src/core/Microsoft.PowerToys.Settings.UI/Views/GeneralPage.xaml

```diff
@@ -40,74 +46,67 @@
+                             IsChecked="{ Binding Mode=TwoWay, Path=IsSystemThemeRadioButtonChecked}"/>
             </muxc:RadioButtons>
 
-            <!--
-                    <TextBlock Text="Default apps" Style="{StaticResource SettingsGroupTitleStyle}"/>
-                <ToggleSwitch Header="Disable telemetry" IsOn="True" Margin="0,14,0,0" />
-                <TextBlock Text="PowerToys currently respects the Windows data &amp; feedback setting" Opacity="0.8" Margin="0,0,0,0" />
-                    -->
-
-            <Button 
-                Content="Restart as admin" 
-                Margin="{StaticResource SmallTopMargin}" 
-                Style="{StaticResource AccentButtonStyle}" 
-                Click="Restart_Elevated" />
-
-            <!--
-                <TextBlock Text="Default apps" Style="{StaticResource SubtitleTextBlockStyle}" Margin="0,34,0,8"/>
-                <ComboBox Header="Shell" SelectedIndex="0" MinWidth="240" Margin="0,14,0,0">
-                    <ComboBoxItem>PowerShell</ComboBoxItem>
-                </ComboBox>
-                <ComboBox Header="Terminal" SelectedIndex="0" MinWidth="240" Margin="0,14,0,0">
-                    <ComboBoxItem>Windows Console</ComboBoxItem>
-                </ComboBox>
-               -->
+            <Button Content="Restart as admin"
+                    Margin="{StaticResource SmallTopMargin}"
+                    Style="{StaticResource AccentButtonStyle}"
+                    Command = "{Binding RestartElevatedButtonEventHandler, Source={StaticResource eventViewModel}}"
+                    />
 
         </StackPanel>
 
-        <StackPanel 
-            x:Name="SidePanel"
-            Orientation="Vertical"
-            HorizontalAlignment="Left"
-            Width="{StaticResource SidePanelWidth}"
-            Grid.Column="1">
-
-            <TextBlock 
-                Text="About PowerToys"
-                Style="{StaticResource SettingsGroupTitleStyle}"
-                Margin="{StaticResource XSmallBottomMargin}"/>
-            <TextBlock 
-                Text="Version 0.15.2" 
-                FontWeight="Bold"
-                Margin="{StaticResource SmallTopMargin}" />
-            <Button 
-                Content="Check for updates" 
-                Margin="{StaticResource SmallTopMargin}" 
-                Style="{StaticResource AccentButtonStyle}" 
-                Click="CheckForUpdates_Click" />
-
-            <HyperlinkButton 
-                Content="Report a bug" 
-                NavigateUri="https://github.com/microsoft/PowerToys/issues" 
-                Margin="{StaticResource SmallTopMargin}" />
-            <HyperlinkButton 
-                Content="Request a feature"  
-                NavigateUri="https://github.com/microsoft/PowerToys/issues"/>
-            <HyperlinkButton 
-                Content="Privacy statement" 
-                NavigateUri="http://go.microsoft.com/fwlink/?LinkId=521839" />
-            <HyperlinkButton 
```

Comment:
Any reason to remove this.

Reply:
I should put this back. Thanks for the catch.